ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py

```diff
@@ -43,3 +124,32 @@ def test_numpy_bincount(
         weights=None,
         minlength=0,
     )
+
+
+@handle_frontend_test(fn_tree="numpy.histogram_bin_edges", data=_histogram_helper())
+def test_numpy_histogram_bin_edges(data):
+    a, bins, range, weights, density = data
+    ret = np_frontend.histogram_bin_edges(
+        a=a,
+        bins=bins,
+        range=range,
+        weights=weights,
+    )
+    ret_gt = np.histogram_bin_edges(
+        a=a,
+        bins=bins,
+        range=range,
+        weights=weights,
+    )
+    assert np.allclose(ret.ivy_array, ret_gt)
+
+
+@handle_frontend_test(fn_tree="numpy.histogram", data=_histogram_helper())
+def test_numpy_histogram(data):
```

Comment:
Hey, we shouldn't  directly use `allclose` in tests. Instead, please following test writing gudies and examples: https://github.com/unifyai/ivy/blob/main/ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py#L53

Reply:
Here is the docs on writing tests for frontend functions: https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html